models/logs.js

```diff
@@ -35,15 +36,17 @@ const addLog = async (type, meta, body) => {
  */
 const fetchLogs = async (query, param) => {
   try {
+    const { dev, ...remainingQuery } = query;
```

Comment:
Parsed dev from the rest of the queries. If dev is true and param is extensionRequests, it will return logs with populated userId